modules/Core/includes/endpoints/CreateReportEndpoint.php

```diff
@@ -79,8 +80,9 @@ public function execute(Nameless2API $api): void {
             'reported_id' => $user_reported_id,
             'report_reason' => $_POST['content'],
             'updated_by' => $user_reporting_data->id,
-            'reported_mcname' => $_POST['reported_username'] ? $_POST['reported_username'] : $reported_user->getDisplayname(),
-            'reported_uuid' => $_POST['reported_uid'] ? $_POST['reported_uid'] : $reported_uuid ?? 'none',
+            'reported_mcname' => $_POST['reported_username'] ?: $reported_user->getDisplayname(),
+            'reported_uuid' => $_POST['reported_uid'] ?: $reported_uuid ?? 'none',
+            'server_id' => $_POST['server_id'],
```

Comment:
The endpoint should probably check if the server id is valid, if one is specified. Of course, not specifying one at all should be allowed for backwards compatibility

Reply:
I can add a check in to prevent a string for example to avoid a MySQL error